packages/console/src/pages/OrganizationRoleDetails/index.tsx

```diff
@@ -54,7 +56,7 @@ function OrganizationRoleDetails() {
     try {
       await api.delete(`api/organization-roles/${data.id}`);
       toast.success(t('organization_role_details.deleted', { name: data.name }));
-      await mutateGlobal('api/roles');
+      await mutateGlobal('api/organization-roles');
```

Comment:
Is this an expected change? Seems irrelavant

Reply:
This should be in another PR.